server/src/Packages/Paypal/Resources/js/types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace GernzyPaypal {
+    type PaypalConfig = {
+        client_id?: string;
```

Comment:
I see this is an optional value here. Would it be possible to make this not optional?

Reply:
cool, did it